cdisc_rules_engine/check_operators/dataframe_operators.py

```diff
@@ -163,14 +163,20 @@ def _check_equality(
         comparison_data = (
             comparator if comparator not in row or value_is_literal else row[comparator]
         )
-        target_values = row[target]
-        target_is_empty = pd.isna(target_values)
-        if not target_is_empty and isinstance(row[target], str):
-            target_is_empty = row[target] == ""
-        comp_is_empty = pd.isna(comparison_data)
-        if not comp_is_empty and isinstance(comparison_data, str):
-            comp_is_empty = comparison_data == ""
-        if target_is_empty or comp_is_empty:
+        # PYTHON PR LOGIC
+        # target_values = row[target]
+        # target_is_empty = pd.isna(target_values)
+        # if not target_is_empty and isinstance(row[target], str):
+        #     target_is_empty = row[target] == ""
+        # comp_is_empty = pd.isna(comparison_data)
+        # if not comp_is_empty and isinstance(comparison_data, str):
+        #     comp_is_empty = comparison_data == ""
+        # if target_is_empty or comp_is_empty:
+        # ORIGINAL LOGIC
+        both_null = (comparison_data == "" or comparison_data is None) & (
+            row[target] == "" or row[target] is None
+        )
+        if both_null:
```

Comment:
@SFJohnson24 The unit tests and test suite are now working with this "original logic" code. Does this mean we should delete the "python pr logic" code, or does it need to work with this instead?